src/rngs/small.rs

```diff
@@ -10,30 +10,37 @@
 
 use rand_core::{Error, RngCore, SeedableRng};
 
-#[cfg(all(not(target_os = "emscripten"), target_pointer_width = "64"))]
-type Rng = rand_pcg::Pcg64Mcg;
-#[cfg(not(all(not(target_os = "emscripten"), target_pointer_width = "64")))]
-type Rng = rand_pcg::Pcg32;
+#[cfg(target_pointer_width = "64")]
+type Rng = super::xoshiro256plusplus::Xoshiro256PlusPlus;
+#[cfg(not(target_pointer_width = "64"))]
+type Rng = super::xoshiro128plusplus::Xoshiro128PlusPlus;
 
 /// A small-state, fast non-crypto PRNG
 ///
 /// `SmallRng` may be a good choice when a PRNG with small state, cheap
 /// initialization, good statistical quality and good performance are required.
-/// It is **not** a good choice when security against prediction or
-/// reproducibility are important.
+/// Note that depending on the application, [`StdRng`] is faster on many modern
+/// platforms while providing higher-quality randomness. Furthermore, `SmallRng`
+/// is **not** a good choice when:
+/// - Security against prediction or reproducibility are important.
+///   Use [`StdRng`] instead.
```

Comment:
This is two separate things: don't recommend `StdRng` for reproducibility!

Reply:
I'll just remove "reproducibility", because this is discussed in detail below.